src/main/kotlin/com/example/noswimmingserver/domain/rank/service/QueryUserRankingService.kt

```diff
@@ -0,0 +1,30 @@
+package com.example.noswimmingserver.domain.rank.service
+
+import com.example.noswimmingserver.domain.rank.domain.repository.UserRankRepository
+import com.example.noswimmingserver.domain.rank.presentation.dto.response.QueryUserRankingElement
+import com.example.noswimmingserver.domain.rank.presentation.dto.response.QueryUserRankingList
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class QueryUserRankingService(
+    private val userRankRepository: UserRankRepository,
+) {
+
+    @Transactional(readOnly = true)
+    fun execute(): QueryUserRankingList {
+        val userRankingList = userRankRepository.queryUserRanking()
+
+        return QueryUserRankingList(
+            userRankingList = userRankingList
+
```

Comment:
🚫 **[ktlint]** <no-blank-lines-in-chained-method-calls> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Needless blank line(s)